Dockerfile

```diff
@@ -1 +1,27 @@
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base_java11:latest
+RUN useradd -r -g root tomcat
```

Comment:
The `tomcat` user should not be in the `root` group, that will give it permissions that are too wide.

Reply:
okay. changed it to tomcat group